Source/Charts/Renderers/YAxisRendererRadarChart.swift

```diff
@@ -12,30 +12,30 @@
 import Foundation
 import CoreGraphics
 
-#if !os(OSX)
-    import UIKit
-#endif
 
 open class YAxisRendererRadarChart: YAxisRenderer
 {
     private weak var chart: RadarChartView?
     
     @objc public init(viewPortHandler: ViewPortHandler, axis: YAxis, chart: RadarChartView)
     {
-        super.init(viewPortHandler: viewPortHandler, axis: axis, transformer: nil)
-        
         self.chart = chart
+
+        super.init(viewPortHandler: viewPortHandler, axis: axis, transformer: nil)
```

Comment:
Initialization in Swift requires the sub class members to be initialized first. Because `chart` is optional, we weren't getting a warning as it was auto initialized to nil.